modules/ossm-installing-the-istioctl-tool.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-installing-the-istioctl-tool_{context}"]
+= Installing the Istioctl tool
+
+.Prerequisites
+
+* You have access to the {ocp-product-title} web console.
+
+.Procedure
+
+. In the {ocp-product-title} web console, click the question mark icon and select *Command Line Tools*.
+
+. Click *Download istioctl*. Choose the version and architecture that matches your system.
+
+* link:https://mirror.openshift.com/pub/cgw/servicemesh/1.23.0/[Version 1.23.0]
+
+* link:https://mirror.openshift.com/pub/cgw/servicemesh/1.24.1/[Version 1.24.1]
+
+. Extract the `istioctl` binary by running the following command:
++
+[source,terminal]
+----
+$ tar xzf istioctl-<VERSION>-<OS>-<ARCH>.tar.gz
+----
+
+. Move to the uncompressed directory by running the following command:
++
+[source,terminal]
+----
+$ cd istioctl-<VERSION>-<OS>-<ARCH>
```

Comment:
After the above step, did not get the directory. 

It is the istioctl file, then executed the below step.  

It workes as expected

Reply:
Correct but for next releases, this parent directory will be present.  

```
$ docker run -it --entrypoint="/bin/bash" istio-must-gather:3.0  -c "tar tvf /opt/istioctl-1.24.2-linux-amd64.tar.gz"
drwxr-xr-x root/root         0 2025-02-18 16:06 istioctl-linux-amd64/
-rwxr-xr-x root/root  96751787 2025-02-18 16:06 istioctl-linux-amd64/istioctl
```